apps/studio/src/features/editing-experience/components/MoveResourceModal/MoveResourceModal.tsx

```diff
@@ -53,23 +53,28 @@ export const MoveResourceModal = () => {
 const MoveResourceContent = withSuspense(
   ({ resourceId, onClose }: { resourceId: string; onClose: () => void }) => {
     // NOTE: This is the stack of user's navigation through the resource tree
-    // the last item here represents his current view
     // NOTE: We should always start the stack from `/` (root)
     // so that the user will see a full overview of their site structure
     const [resourceStack, setResourceStack] = useState<PendingMoveResource[]>(
       [],
     )
-    const moveDest = resourceStack[resourceStack.length - 1]
+    const [isResourceHighlighted, setIsResourceHighlighted] =
+      useState<boolean>(true)
     const { siteId } = useQueryParse(sitePageSchema)
     const setMovedItem = useSetAtom(moveResourceAtom)
     const [{ title }] = trpc.resource.getMetadataById.useSuspenseQuery({
       resourceId,
     })
-    const curResourceId = resourceStack[resourceStack.length - 1]?.resourceId
+    const moveDest = resourceStack[resourceStack.length - 1]
+    const parentDest = resourceStack[resourceStack.length - 2]
+    const curResourceId = moveDest?.resourceId
```

Comment:
tbvh i think at this point we should make this into a custom hook cos there's alot of logic, but i'm ok with merging this first - more important to get it working than to aim for perfection